static/sass/utils/_buttons.scss

```diff
@@ -60,6 +71,7 @@
 // dark buttons in mobile nav are a bit shorter so nav doesn't change height on scroll
 $button-nav-height: 24px;
 $button-default-height: 26px;
+$button-large-height: 44px;
```

Comment:
do these match up with the form input heights? perhaps we could standardize them

Reply:
Yes, this matches with the form heights. I agree about standardizing. I was nervous to update everything since it affects so many pages. I think we should refactor after we have killed off some of the extra routes.